packages/commonwealth/client/scripts/views/pages/MyCommunityStake/Transactions/Transactions.tsx

```diff
@@ -87,14 +92,51 @@ const Transactions = ({ filterOptions }: TransactionsProps) => {
         columnInfo={columnInfo}
         rowData={filteredTransactionHistoryData.map((tx) => ({
```

Comment:
do we have a plan to add pagination here?

Reply:
i didnt find those in the UI/Figmae, but u r right, we will eventually have those, will discuss with team if we have designs for those